packages/terra-search-field/tests/wdio/search-field-spec.js

```diff
@@ -156,6 +190,8 @@ Terra.describeViewports('Search Field', ['medium'], () => {
     });
 
     it('should display Search Field with long enough text', () => {
+      // wait for the default search delay time for callback text to update
+      browser.pause(2500);
```

Comment:
Is this delay needed? there doesn't seem to be logic executing before this.

Reply:
Is this needed for every test or can we just do it once in the before block?

Reply:
Well, it is needed only when we are checking for search text to be updated/reflected in some tests. It is not required before all the tests since not all tests are checking for search text updates. I did not add in a beforeAll because didn't want to delay tests unnecessarily. 
Here are some example snapshots we validate where we need to pause for search text updates to reflect:
https://github.com/cerner/terra-core/blob/main/packages/terra-search-field/tests/wdio/__snapshots__/reference/terra-default-theme/en/chrome_medium/search-field-spec/callback_with_too_short_text.png 
https://github.com/cerner/terra-core/blob/main/packages/terra-search-field/tests/wdio/__snapshots__/reference/terra-default-theme/en/chrome_medium/search-field-spec/callback_with_long_enough_text.png

Reply:
Concurring with @sdadn and a question: Is it actually possible to avoid doing things like this? https://webdriver.io/docs/api/element/isFocused/

I'm seeing in an example that we do:
```
const loginInput = await $('[name="login"]');
console.log(await loginInput.isFocused()); // outputs: false
```

Could we attempt to do something similar here?

Reply:
@kenk2 async methods were introduced in v8 I believe. We're using v7.

https://v7.webdriver.io/docs/api/element/isFocused

Reply:
@nikhitasharma thanks for answering my question. That's fair and is fine by me. 👍